library/Fission/AWS/Route53.hs

```diff
@@ -1,147 +1,3 @@
-module Fission.AWS.Route53
-  ( createChangeRequest
-  , registerDomain
-  ) where
+module Fission.AWS.Route53 (module Fission.AWS.Route53.Class) where
```

Comment:
why bother with this instead of just eliminating this module entirely? readability in imports?

Reply:
That, and because in the future a lot of functionality could live in this module. Having to run around and remove the prefix to get access to helper functions could be frustrating. Probably not a big deal in either case, so happy to delete these if it's helpful.

Reply:
nope just curious :+1: